openslides_backend/permissions/permission_helper.py

```diff
@@ -79,3 +79,40 @@ def is_child_permission(child: Permission, parent: Permission) -> bool:
         parents = permission_parents[current]
         queue.extend(parents)
     return False
+
+
+def has_organisation_management_level(
+    datastore: DatastoreService, user_id: int, set_level: OrganisationManagementLevel
+) -> bool:
+    """ Checks wether a user has the minimum necessary OrganisationManagementLevel """
+    hierarchy: Dict[str, int] = {
+        OrganisationManagementLevel.SUPERADMIN: 3,
+        OrganisationManagementLevel.CAN_MANAGE_ORGANISATION: 2,
+        OrganisationManagementLevel.CAN_MANAGE_USERS: 1,
+    }
+
+    if user_id > 0:
+        user = datastore.get(
+            FullQualifiedId(Collection("user"), user_id),
+            ["organisation_management_level"],
+        )
+        set_level_number = hierarchy.get(set_level, 4)
+        actual_level_number = hierarchy.get(
+            user.get("organisation_management_level"), 0  # type: ignore
+        )
+        if actual_level_number >= set_level_number:
+            return True
+    return False
+
+
+def is_temporary(datastore: DatastoreService, instance: Dict[str, Any]) -> bool:
+    """
+    Checks whether the user, identified by the id the instance, is a temporary user.
+    Be carefull about the stored meeting id in the instance!
+    """
+    if "meeting_id" not in instance:
+        db_instance = datastore.get(
+            FullQualifiedId(Collection("user"), instance["id"]), ["meeting_id"]
+        )
+        instance["meeting_id"] = db_instance.get("meeting_id")
```

Comment:
You are potentially altering the action data here, please remove

Reply:
We talked about and decided to let the meeting_id in the instance